b1ddi/schema_dns_data_record_rdata_test.go

```diff
@@ -0,0 +1,131 @@
+package b1ddi
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_UpdateDataRecordRData(t *testing.T) {
+	testData := map[string]struct {
+		input       map[string]interface{}
+		output      map[string]interface{}
+		recordType  string
+		errExpected bool
```

Comment:
I'm wondering if it shouldn't be an integer containing the number of expected errors. This way for `SRV` negative scenario where we have 3 errors, we can check that the errors are accumulating.

Reply:
Sure. I will update the UT. I did test using wrong values for both option and rData and could see accumulated errors showing up.

```

b1ddi_dns_auth_zone.tf_example_auth_zone: Destroying... [id=dns/auth_zone/29dc94d8-b406-4149-a61e-33edb123229d]
b1ddi_dns_record.srv_test: Creating...
b1ddi_dns_auth_zone.tf_example_auth_zone: Destruction complete after 1s
╷
│ Error: unable to parse key 'port': strconv.Atoi: parsing "1688qqwe": invalid syntax
│
│   with b1ddi_dns_record.srv_test,
│   on resource-test.tf line 39, in resource "b1ddi_dns_record" "srv_test":
│   39: resource "b1ddi_dns_record" "srv_test" {
│
╵
╷
│ Error: unable to parse key 'priority': strconv.Atoi: parsing "0123po": invalid syntax
│
│   with b1ddi_dns_record.srv_test,
│   on resource-test.tf line 39, in resource "b1ddi_dns_record" "srv_test":
│   39: resource "b1ddi_dns_record" "srv_test" {
│
╵
╷
│ Error: unable to parse key 'weight': strconv.Atoi: parsing "lop50": invalid syntax
│
│   with b1ddi_dns_record.srv_test,
│   on resource-test.tf line 39, in resource "b1ddi_dns_record" "srv_test":
│   39: resource "b1ddi_dns_record" "srv_test" {
│